pkg/plugins/policies/meshaccesslog/plugin/v1alpha1/plugin.go

```diff
@@ -24,7 +31,78 @@ func (p plugin) MatchedPolicies(dataplane *core_mesh.DataplaneResource, resource
 	return matchers.MatchedPolicies(api.MeshAccessLogType, dataplane, resources)
 }
 
+func (p plugin) applyToOutbounds(rs *core_xds.ResourceSet, proxy *core_xds.Proxy) error {
+	logPolicies, ok := proxy.Policies.Dynamic[api.MeshAccessLogType]
+	if !ok {
+		return nil
+	}
+
+	sourceService := proxy.Dataplane.Spec.GetIdentifyingService()
+
+	outboundListeners := map[mesh_proto.OutboundInterface]*envoy_listener.Listener{}
+	for _, res := range rs.Resources(envoy_resource.ListenerType) {
+		if res.Origin != generator.OriginOutbound {
+			continue
+		}
+		listener := res.Resource.(*envoy_listener.Listener)
+
+		address := listener.GetAddress().GetSocketAddress()
+
+		outboundListeners[mesh_proto.OutboundInterface{
+			DataplaneIP:   address.GetAddress(),
+			DataplanePort: address.GetPortValue(),
+		}] = listener
+	}
+
+	for _, outbound := range proxy.Dataplane.Spec.Networking.GetOutbound() {
+		oface := proxy.Dataplane.Spec.Networking.ToOutboundInterface(outbound)
+
+		listener, ok := outboundListeners[oface]
+		if !ok {
+			continue
+		}
+
+		serviceName := outbound.GetTagsIncludingLegacy()[mesh_proto.ServiceTag]
+
+		var toPolicy *api.MeshAccessLog_Conf
+		for _, toRule := range logPolicies.ToRules.Rules {
```

Comment:
you can use:
```go
toPolicy = logPolicies.ToRules.Rules.Compute(Subset{
    {Key: mesh_proto.ServiceTag, Value: serviceName},
}).(*api.MeshAccessLog_Conf)
```
to get conf for the outbound

Reply:
Still need to check if it's nil, right? If no `to` rules match.